### ISSUE TITLE
UndefVarError: UpTri not defined

### ISSUE THREAD

**Issue by jebej** [JuliaSIMD/LoopVectorization.jl#367]
In a clean environment with only LoopVectorization v0.12.98, on Windows and julia 1.6.3, precompilation fails with the following error:
```
julia> using LoopVectorization
[ Info: Precompiling LoopVectorization [bdcacae8-1622-11e9-2a5c-532679323890]
ERROR: LoadError: UndefVarError: UpTri not defined
Stacktrace:
 [1] include
   @ .\Base.jl:386 [inlined]
 [2] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)
   @ Base .\loading.jl:1235
 [3] top-level scope
   @ none:1
 [4] eval
   @ .\boot.jl:360 [inlined]
 [5] eval(x::Expr)
   @ Base.MainInclude .\client.jl:446
 [6] top-level scope
   @ none:1
in expression starting at C:\Users\jebej\.julia\packages\LoopVectorization\x4G96\src\LoopVectorization.jl:1
ERROR: Failed to precompile LoopVectorization [bdcacae8-1622-11e9-2a5c-532679323890] to C:\Users\jebej\.julia\compiled\v1.6\LoopVectorization\jl_2588.tmp.
Stacktrace:
 [1] error(s::String)
   @ Base .\error.jl:33
 [2] compilecache(pkg::Base.PkgId, path::String, internal_stderr::Base.TTY, internal_stdout::Base.TTY, ignore_loaded_modules::Bool)
   @ Base .\loading.jl:1385
 [3] compilecache(pkg::Base.PkgId, path::String)
   @ Base .\loading.jl:1329
 [4] _require(pkg::Base.PkgId)
   @ Base .\loading.jl:1043
 [5] require(uuidkey::Base.PkgId)
   @ Base .\loading.jl:936
 [6] require(into::Module, mod::Symbol)
   @ Base .\loading.jl:923
```

**Comment by jebej** [comment]
Looks like `UpTri` and `LowTri` were removed in `ArrayInterface` 3.2:
https://github.com/JuliaArrays/ArrayInterface.jl/compare/v3.1.40...v3.2.0

@Tokazama We could probably get these two back in since a minor version doesn't need to remove trivial definitions like that.

**Comment by Tokazama** [comment]
On it

**Comment by jebej** [comment]
I can confirm that this is fixed on v3.2.1. Maybe we should add a version exclusion in the project file here, otherwise feel free to close.

**Comment by chriselrod** [comment]
LoopVectorization's Project.toml currently doesn't allow for v.3.2.0 at all, so I'm not quite sure how you were using that combination in the first place. But I'm adding:
https://github.com/JuliaSIMD/LoopVectorization.jl/commit/ce5436bbf99f760addfb5e24bda66e99fe6b0779
